src/hooks/query/loginQuery.js

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from 'react-query';
+import { fetchUsers } from '../../services/userService';
+
+const LOGIN_USER = 'LOGIN_USER';
+export const useQueryToLogin = (config) => {
```

Comment:
In this query you are receiving users data, so, make a query name to be understandable what is happening here

Reply:
take a look above please, again.
and also add a default value for config here in case if you provide nothing. (as I showed you, config={})